airbyte-integrations/connectors/source-notion/source_notion/schemas/shared/title_text.json

```diff
@@ -0,0 +1,59 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": true,
+  "properties": {
```

Comment:
I'm referencing the same schema in two places in `title.json`, to avoid repetition and potential divergence of those two places, I'm using a separate file.